core/domain/subtopic_page_services_test.py

```diff
@@ -237,25 +226,118 @@ def test_migrate_page_contents_to_latest_schema(self):
         current_schema_version_swap = self.swap(
             feconf, 'CURRENT_SUBTOPIC_PAGE_CONTENTS_SCHEMA_VERSION', 2)
         subtopic_page_swap = self.swap(
-            subtopic_page_domain, 'SubtopicPage', MockSubtopicPage)
-
-        subtopic_page_model = topic_models.SubtopicPageModel.get(
-            self.subtopic_page_id)
+            subtopic_page_domain, 'SubtopicPage',
+            subtopic_page_domain.SubtopicPage)
+        html = (
+            '<p>Value</p><oppia-noninteractive-math ' +
```

Comment:
We don't need the `+` here right? (Ditto everywhere in thsi PR!)

Reply:
We need it because otherwise, the type of the variable will become tuple rather than a Unicode string.

Reply:
I don't think so, have you tried removing the `+`? [Have you checked the existing multiline string in the codebase?]

Reply:
I don't think so, have you tried removing the `+`? [Have you checked the existing multiline string in the codebase?]

Reply:
done, removed it.